### PR TITLE
feat: implement missing month theme

### PR DESCRIPTION
**Pull request by adhrinae**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)
- [x] It does not introduce a breaking change or has description for the breaking change

### Description

- Implemented missing options in theme
  - `month.moreView.height` & `month.moreView.width`
  - `month.gridCell`

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by dotaitch — APPROVED** [review]
리뷰 완료합니다.

File: apps/calendar/docs/ko/apis/theme.stories.mdx [hunk 1]
```diff
@@ -186,22 +186,29 @@ interface MonthTheme {
...
+    width: number | null,
+    height: number | null,
   };
   moreViewTitle: {
     backgroundColor: string;
   };
   weekend: { backgroundColor: string };
+  gridCell: {
+    headerHeight: number | null;
+    footerHeight: number | null;
```
Comment (lja1018) [inline apps/calendar/docs/ko/apis/theme.stories.mdx:198]: 다른 css 프로퍼티 값처럼 `string`으로 받는게 낫지 않을까요?

**Review by lja1018 — COMMENTED** [review]
리뷰 완료합니다.

File: apps/calendar/docs/ko/apis/theme.stories.mdx (hunk 1 above)
Comment (adhrinae) [reply apps/calendar/docs/ko/apis/theme.stories.mdx:198]: 처음엔 그러려고 했는데 이 숫자값이 레이아웃 계산에 쓰이고 있어서 string으로는 못받겠더군요.

Comment (adhrinae) [reply apps/calendar/docs/ko/apis/theme.stories.mdx:198]: https://github.com/nhn/tui.calendar/pull/1139/files#diff-e3515bfcc50fedb9d81814c966a85be738901efcf6047832aad00bc5d352c45eR40-R58

**Pull request merged**